client/src/featureform/search.py

```diff
@@ -0,0 +1,26 @@
+import os
+import requests
+from .format import *
+
+
+def search(phrase):
+    host = os.getenv('FEATUREFORM_HOST')
```

Comment:
@sdreyer, there is a possible issue here that is somewhat novel: Given all other methods in the resource client use gRPC, which means they rely on the check in the constructor and then `self._stub`, we don't save the value of `host` on the RC instance. If a user is simply passing the `--host` flag on the command line, this won't work. They need to export the value in the session instead.

Are there any objections to storing `host` on the RC instance?

Reply:
Storing it is fine